js/common/.swcrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Why would you want to build it?

I guess unless we want to publish the package, it's easier to allow it being built by the application that is using it - it mitigates the need to build it manually during project setups or in case of made changes.

We have applied such approach at dApps repo, and so far it has been extremely comfortable to use.